python/pyspark/sql/tests/connect/test_parity_functions.py

```diff
@@ -34,6 +34,14 @@ def test_function_parity(self):
     def test_input_file_name_reset_for_rdd(self):
         super().test_input_file_name_reset_for_rdd()
 
+    @unittest.skip("SPARK-50050: Spark Connect should support str ndarray.")
+    def test_str_ndarray(self):
+        super().test_str_ndarray()
+
+    @unittest.skip("SPARK-50051: Spark Connect should empty ndarray.")
+    def test_empty_ndarray(self):
+        super().test_empty_ndarray()
```

Comment:
existing connect side implementation is based on `array` function, so not support empty ndarray, needs more investigation.